developer-hub/src/polkadot_sdk/mod.rs

```diff
@@ -0,0 +1,126 @@
+//!
+//! [`cumulus`] transforms FRAME-based runtimes into Polkadot-compatible parachain runtimes, and
+//! substrate-based client into Polkadot-compatible collators.
+//!
+//! #### Polkadot
+//!
+//! [![Polkadot-license](https://img.shields.io/badge/License-GPL3-blue)](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/LICENSE)
+//! [![GitHub Repo](https://img.shields.io/badge/github-polkadot-e6007a?logo=polkadot)](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot)
+//!
+//! [`polkadot`] is an implementation of a Polkadot client in Rust, by `@paritytech`. The Polkadot
+//! runtimes are located under the [`fellowship/runtime`](https://github.com/polkadot-fellows/runtimes) repository.
+//!
+//! > [`polkadot`] contains useful links to further learn about Polkadot, **but is in general not
+//! > part of the SDK**, as it is rarely used by developers who wish to build on top of Polkadot.
+//!
+//! #### XCM
+//!
+//! [![XCM-license](https://img.shields.io/badge/License-GPL3-blue)](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/LICENSE)
+//! [![GitHub Repo](https://img.shields.io/badge/github-XCM-e6007a?logo=polkadot)](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/xcm)
+//!
+//! [`xcm`], short for "cross consensus message", is the primary format that is used for
+//! communication between parachains, but is intended to be extensible to other use cases as well.
+//!
+//! ### Summary
+//!
+//! The following diagram summarizes how components of the Polkadot-SDK work together:
+#![doc = simple_mermaid::mermaid!("../../../docs/mermaid/polkadot_sdk.mmd")]
+//!
+//! 1. A Substrate-based chain is a blockchain composed of a "Runtime" and a "Client". As noted
+//!    above, the "Runtime" is the application logic of the blockchain, and the "Client" is
+//!    everything else. See [`reference_docs::wasm_meta_protocol`] for an in-depth explanation of
+//!    this. The former is built with [`frame`], and the latter is built with Substrate client
+//!    libraries.
+//! 2. Polkadot is itself a Substrate-based chain, composed of the the exact same two components.
+//!    The Polkadot client code is in [`polkadot`], and the Polkadot runtimes are controlled by the
+//!    [Polkadot Fellowship](https://github.com/polkadot-fellows).
+//! 3. A parachain is a "special" Substrate based chain, whereby both the client and the runtime
+//!    components have became "Polkadot-aware" using Cumulus.
+//!
+//! ## History
+//!
+//! Substrate, Polkadot and Cumulus used to each have their own repository, each of which is now
+//! archived. For historical context about how they merged into this mono-repo, see:
+//!
+//! - <https://polkadot-public.notion.site/Polkadot-SDK-FAQ-fbc4cecc2c46443fb37b9eeec2f0d85f>
+//! - <https://forum.polkadot.network/t/psa-parity-is-currently-working-on-merging-the-polkadot-stack-repositories-into-one-single-repository/2883>
+//!
```

Comment:
I can't immediately see how this is helpful information. Unless there's some something I'm missing, I'd personally just remove it.

Reply:
I would keep this "history" section. I could mean something for people who accidentally found some old documentation and is now becoming aware of the monorepo

Reply:
Meet in the middle, as in, we remove it in eg. 6 months? I feel like atm there is som many links to the new archived repos that this might be useful as reference. Although, the README of the archived repos is pretty clear on what has happened. 

UPDATE: I think I will remove it in the end :)

Reply:
Removed.